libass/ass_outline.c

```diff
@@ -113,19 +233,33 @@ void outline_free(ASS_Outline *outline)
 /*
  * \brief Add a single point to a contour.
```

Comment:
Perhaps add a comment here noting that a segment is added (if and) only if the `segment` argument is nonzero.